src/geneweaver/api/services/genes.py

```diff
@@ -72,3 +72,32 @@ def get_gene_mapping(
         raise err
 
     return {"ids_map": ids_map}
+
+
+def get_gene_aon_mapping(
+    cursor: Cursor,
+    source_ids: List[str],
+    target_species: Species,
```

Comment:
Same as in PR 32. This is just the species. This endpoint only work within a species, so the source and target are the same.